publication-commons/src/main/java/no/unit/nva/publication/rightsretention/RightsRetentionsValueFinder.java

```diff
@@ -119,7 +119,7 @@ private RightsRetentionStrategy getRrsForUnPublishedFile(File file,
                 throw new IllegalArgumentException("Unknown RightsRetentionStrategy type " + fileRightsRetention);
         };
         if (!isValid(rrs)) {
-            throw new BadRequestException(ILLEGAL_RIGHTS_RETENTION_STRATEGY_ON_FILE + file.getIdentifier());
+            throw new BadRequestException(ILLEGAL_RIGHTS_RETENTION_STRATEGY_ON_FILE + file.getIdentifier() + fileRightsRetention);
```

Comment:
Space between file identifier and `fileRightsRetention`? Might want to change this to `String.format`